src/igvfd/schemas/treatment.json

```diff
@@ -0,0 +1,206 @@
+{
+    "title": "Treatment",
+    "id": "/profiles/treatment.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
```

Comment:
I am not sure this is a reasonable list of required properties. I am not sure duration should be required. If the amount is required then the units should be required as well, otherwise, you are leaving it to the user to double guess why their submission is not valid, and require them to read dependencies?